### ISSUE TITLE
Make sure plans are executed on the correct date

### ISSUE THREAD

**Issue by mtotschnig** [mtotschnig/MyExpenses#448]
Currently plans are executed every six hours and take into account an exact time stamp. This leads to a situation where plans whose date is set late in the evening, will eventually be executed only early the next day.
The fix should consist in executing plans only once per day (or whenever a new plan is defined) and always take into account all plans of the respective day irrespective of their exact time.
